java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -2558,6 +2559,93 @@ public void zlexcount_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void zrangestore_by_lex_returns_success() {
+        // setup
+        String source = "testSourceKey";
```

Comment:
With a fresh look I see it is an overkill for testing one command implemented by 2 functions.
Consider to create a set of UT for rangeoptions and reduce commands' UT. (in another PR)
The same applicable for zaddoptions and all others.